unityfiles/Assets/Scripts/GUI/LevelTimeToGuiHandler.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LevelTimeToGuiHandler : MonoBehaviour{
+	
+	private TimerGUI timerGUI;
+
+	private LevelManager levelManager;
+
+	private void Start() {
+		if (TryGetComponent(out TimerGUI timerGuiSript)){
+			this.timerGUI = timerGuiSript;
+		}else{
+			Debug.LogError("Cant find timer GUI");
+		}
+	}
+	
+	private void Update() {
+		this.timerGUI.SetTime(LevelManager.Instance.GetLevelTimeLeft());
+	}
```

Comment:
Should it be a `Singleton` ?

Reply:
Nei. Andre skal ikke ha behov for å snakke elns med denne. og gjør ikke noe om det er flere av denne.